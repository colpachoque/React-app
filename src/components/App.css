body {
  margin: 0;
}

.container {
  min-height: 74vh;
}
