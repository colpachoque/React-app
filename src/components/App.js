import React from 'react';
import Header from './Header';
import Footer from './Footer';
import MainPage from './MainPage';

const movies = [
    {
        name: 'Kill Bill',
        genre: 'Action and adventure',
        year: 2004
    },
    {
        name: 'Kill Bill',
        genre: 'Action and adventure',
        year: 2004
    },
    {
        name: 'Kill Bill',
        genre: 'Action and adventure',
        year: 2004
    },
    {
        name: 'Kill Bill',
        genre: 'Action and adventure',
        year: 2004
    },
    {
        name: 'Kill Bill',
        genre: 'Action and adventure',
        year: 2004
    },
    {
        name: 'Kill Bill',
        genre: 'Action and adventure',
        year: 2004
    },
    {
        name: 'Kill Bill',
        genre: 'Action and adventure',
        year: 2004
    }
];
let App = () => (
    <>
        <Header/>
            <MainPage movies={movies}/>
        <Footer/>
    </>);

export default App;