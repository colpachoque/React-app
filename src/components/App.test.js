import React from "react";
import App from "./App";
import { shallow, mount } from "enzyme";
import toJson from "enzyme-to-json";
describe("App component", () => {
  it("should be render correctly", () => {
    const component = shallow(<App />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
