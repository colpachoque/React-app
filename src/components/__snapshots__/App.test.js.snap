// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should be render correctly 1`] = `
<Fragment>
  <ErrorBoundary>
    <Header
      onSearch={[Function]}
    />
    <MainPage
      search=""
    />
    <Footer />
  </ErrorBoundary>
</Fragment>
`;
